CONTRIBUTING.md

```diff
@@ -25,8 +25,16 @@ If you want to contribute to the projects that are maintained on github, there a
 
 > 1. Identified one of the issues above that you feel you can contribute to, you're ready to make a change to the project repository!
 > 2. **[Fork] the repository**. This makes your own version of this project you can edit and use.
-> 3. **Make your changes**! You can do this in the GitHub interface on your own local machine. Once you're happy with your changes...
-> 3. **Submit a pull request**. This opens a discussion around your project and lets the project lead know you are proposing changes.
+> 3. **Make your changes**! You can do this in the GitHub interface on your own local machine\*. Once you're happy with your changes...
+> 4. **Submit a pull request**. This opens a discussion around your project and lets the project lead know you are proposing changes.
+
+\* To preview the website on your own machine, you will need to install [Jekyll](https://jekyllrb.com/) and:
+
+```
+git clone git@github.com:resbazaz/studyGroup.git
+cd studyGroup
+jekyll serve
+```
```

Comment:
Last time I did this, I remember encountering huge problems... but it's working now and I don't remember what I did to solve it? Can anyone try this and lmk if it breaks?

FWIW, I seem to remember something about needing to install an old version of Jekyll; I'm currently using 4.2.2

Reply:
This worked for me *after* trying to install jekyll 2 different ways.  The official method [here](https://jekyllrb.com/docs/installation/macos/) did not work for me on macOS with several steps being broken.  The suggestions in troubleshooting were helpful (https://jekyllrb.com/docs/troubleshooting/#installation-problems).  I ended up installing ruby with homebrew, then installing jekyll with ruby, then running jekyl serve.